html/reaction.html

```
@@ -254,11 +254,24 @@
                     </span>
                   </legend>
                   <div class="features"></div>
-                  <div onclick="ord.features.add($(this).closest('.component'));" class="add">+ add feature</div>
+                  <div onclick="ord.compounds.addFeature($(this).closest('.component'));" class="add">+ add feature</div>
                 </fieldset>
               </fieldset>
             </div>
 
+            <div id="feature_template" class="feature" style="display: none;">
```

This is more a comment on the data_template than the feature_template, but if you select "number" as the type and enter a non-parseable float, it will fail silently and just exclude that value from the proto. Perhaps we need to add a `floattext` class to the editable div when "number" is selected as we did with other float fields?

---

Done.